CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Changed
+- Changed syntax: `--envfile <filename>` is now a required parameter
```

Comment:
Generally, I try to skip the first "Changed" here in "Changed syntax" since we are already under the "Changed" headline. Makes it a bit shorter and direct :smiley_cat:

Reply:
ok!